slides/slides.md

```diff
@@ -75,8 +105,72 @@ class: dense
 - Great for POCs
 - Need to build full app for distribution
 
+<div class="flex flex-col absolute bottom-8 right-6" style="width: 200px;">
+Please install Expo app
+👇 Android or iOS 👇
+<img src="/images/qrcode_expo.dev.png" style="height: 200px; width: 200px; margin: 10px 0;">
+https://expo.dev/client
+</div>
+
 ---
 
 # Start here
 
-https://reactnative.dev/docs/environment-setup
+- Clone this workshop!
+  1. https://github.com/nearform/react-native-workshop
+  2. Open these slides from the readme
+  3. Look in `./step1` etc for code and more info in `README`s
+
+- React Native's quickstart guide:
+
+https://reactnative.dev/docs/environment-setup?guide=quickstart
+
+<div class="flex flex-col absolute bottom-8 right-6" style="width: 200px;">
+Please install Expo app
+👇 Android or iOS 👇
+<img src="/images/qrcode_expo.dev.png" style="height: 200px; width: 200px; margin: 10px 0;">
+https://expo.dev/client
+</div>
+  
+---
+
+# [Step 0: Hello world](https://github.com/nearform/react-native-workshop/tree/main/step0)
+
+- Create and run starter Expo app:
+  - `npx create-expo-app NodeConfGame`
+    - (want TypeScript not JS? Add `-t`)
```

Comment:
I think that flag also requires a value for the template to use

I used `npx create-expo-app -t expo-template-blank-typescript` when creating the original project.